api/app/src/external/commonwell/document/document-query.ts

```diff
@@ -92,24 +85,9 @@ export async function queryAndProcessDocuments({
         documents: cwDocuments,
         override,
       });
-      log(`Downloaded and upserted ${fhirDocRefs.length} docs`);
 
       reportDocQueryUsage(patient);
 
-      // send webhook to cx async when docs are done downloading
-      processPatientDocumentRequest(
-        organization.cxId,
-        patient.id,
-        MAPIWebhookType.documentDownload,
-        MAPIWebhookStatus.completed,
-        toDTO(fhirDocRefs)
-      );
-
-      await updateDocQuery({
-        patient: { id: patient.id, cxId: patient.cxId },
-        downloadProgress: { status: "completed" },
-      });
```

Comment:
Moved to the function that determines status (`downloadDocsAndUpsertFHIR`), better encapsulated.